src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -91,18 +86,14 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     )
   }
 
-  private val submitter = new Submitter(
-    getConfiguredDirectory("deposits.stage-for-submit"),
-    submitBase,
-    configuration.properties.getString("deposit.permissions.group"),
-  )
+  private val submitter = new Submitter(stagedDir, submitBase, properties.getString("deposit.permissions.group"))
```

Comment:
would it make sense to use named arguments here?

```suggestion
  private val submitter = new Submitter(
      stagingBaseDir = stagedDir,
      submitToBaseDir = submitBase,
      groupName = properties.getString("deposit.permissions.group"),
  )
```

Not sure though... depends on readability perhaps